content/agent/basic_agent_usage/heroku.md

```diff
@@ -101,7 +101,7 @@ heroku drains:add https://http-intake.logs.datadoghq.eu/v1/input/<API_KEY>?ddsou
 
 * Replace `<API_KEY>` with your [Datadog API Key][2]
 * Replace `<APPLICATION_NAME>` and `<SERVICE>` with your application name
-* Replace `<HOST>` by the wanted hostname. Refer to the [host section](#hostname) to see how it is set for metrics. Note that it is not yet possible to set the dyno name as the hostname for logs.
+* Replace `<HOST>` by the wanted hostname. Refer to the [host section](#hostname) to see how it is set for metrics. Note that it is not yet possible to dynamically set the dyno name as the hostname for logs.
```

Comment:
" Replace `<HOST>` with the desired hostname. Note that per [host section](#hostname) metrics and traces will set the default host name to dyno name. It is not yet possible to dynamically set the dyno name as the hostname for logs.  In the mean time to correlate between metrics, traces and logs the ____ tag can be used.

Reply:
___ can be filled with `dyno` and `dynotype`